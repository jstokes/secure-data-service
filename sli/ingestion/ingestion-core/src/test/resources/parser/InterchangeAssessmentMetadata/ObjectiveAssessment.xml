<?xml version="1.0" encoding="UTF-8"?>
<InterchangeAssessmentMetadata xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <ObjectiveAssessment id="ID001">
    <IdentificationCode>HJDowIqe5ynYf-a</IdentificationCode>
    <MaxRawScore>-616777533</MaxRawScore>
    <AssessmentPerformanceLevel>
      <PerformanceLevel id="ID003" ref="ID002">
        <CodeValue>Wsv8Xq4AlZ_JQFnv</CodeValue>
        <PerformanceLevelMet>true</PerformanceLevelMet>
      </PerformanceLevel>
      <AssessmentReportingMethod>Z-score</AssessmentReportingMethod>
      <MinimumScore>-198225122</MinimumScore>
      <MaximumScore>273522200</MaximumScore>
    </AssessmentPerformanceLevel>
    <PercentOfAssessment>-379146788</PercentOfAssessment>
    <Nomenclature>J</Nomenclature>
    <AssessmentItemReference>
      <AssessmentItemIdentity>
        <AssessmentItemIdentificationCode>IhKDMMqx5G6S5LVQCerE3</AssessmentItemIdentificationCode>
      </AssessmentItemIdentity>
    </AssessmentItemReference>
    <LearningObjectiveReference>
      <LearningObjectiveIdentity>
        <Objective>piv</Objective>
        <AcademicSubject>Communication and Audio/Visual Technology</AcademicSubject>
        <ObjectiveGradeLevel>Preschool/Prekindergarten</ObjectiveGradeLevel>
      </LearningObjectiveIdentity>
    </LearningObjectiveReference>
    <LearningStandardReference>
      <LearningStandardIdentity>
        <IdentificationCode>HTDJdlX-7HbFcSUxFiBPLj6iODRUFJDSrsdKLkR6H-DeyW-9</IdentificationCode>
      </LearningStandardIdentity>
    </LearningStandardReference>
    <ObjectiveAssessmentReference>
      <ObjectiveAssessmentIdentity>
        <ObjectiveAssessmentIdentificationCode>VZDSaOEQ6c5j2J.taMsH15xtlDp_OuPqujkxHwkLHQyrSG</ObjectiveAssessmentIdentificationCode>
      </ObjectiveAssessmentIdentity>
    </ObjectiveAssessmentReference>
        <AssessmentReference>
      <AssessmentIdentity>
        <AssessmentTitle>vj8zbHeMTeYFobCU0TF21q-Cy8n</AssessmentTitle>
        <AcademicSubject>Hospitality and Tourism</AcademicSubject>
        <GradeLevelAssessed>Tenth grade</GradeLevelAssessed>
        <Version>-1179495731</Version>
      </AssessmentIdentity>
    </AssessmentReference>
  </ObjectiveAssessment>
</InterchangeAssessmentMetadata>
