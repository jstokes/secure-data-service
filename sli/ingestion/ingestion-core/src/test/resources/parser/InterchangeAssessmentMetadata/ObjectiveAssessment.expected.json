{
  "a_id":"ID001",
  "IdentificationCode":{"_value":"HJDowIqe5ynYf-a"},
  "MaxRawScore":-616777533,
  "AssessmentPerformanceLevel":[{
    "PerformanceLevel":{
      "a_id":"ID003",
      "a_ref":"ID002",
      "CodeValue":{"_value":"Wsv8Xq4AlZ_JQFnv"},
      "PerformanceLevelMet":true
    },
    "AssessmentReportingMethod":{"_value":"Z-score"},
    "MinimumScore":-198225122,
    "MaximumScore":273522200
  }],
  "PercentOfAssessment":-379146788,
  "Nomenclature":{"_value":"J"},
  "AssessmentItemReference":[{
    "a_id":"ID005",
    "a_ref":"ID002"
  }],
  "LearningObjectiveReference":[{
      "LearningObjectiveIdentity":{
        "Objective":{"_value":"piv"},
        "AcademicSubject":{"_value":"Communication and Audio/Visual Technology"},
        "ObjectiveGradeLevel":{"_value":"Preschool/Prekindergarten"}
      }
  }],
  "LearningStandardReference":[{
    "LearningStandardIdentity":{
      "IdentificationCode":{"_value":"HTDJdlX-7HbFcSUxFiBPLj6iODRUFJDSrsdKLkR6H-DeyW-9"}
    }
  }],
  "ObjectiveAssessmentReference":[{
    "ObjectiveAssessmentIdentity":{
      "ObjectiveAssessmentIdentificationCode":{"_value":"VZDSaOEQ6c5j2J.taMsH15xtlDp_OuPqujkxHwkLHQyrSG"}
    }
  }]
}